### PR TITLE
{2023.06}[gompi/2023a] BLAST+ v2.14.1

### PR DESCRIPTION
**Pull request by TopRichard**
```
2 out of 39 required modules missing:

* cpio/2.15-GCCcore-12.3.0 (cpio-2.15-GCCcore-12.3.0.eb)
* BLAST+/2.14.1-gompi-2023a (BLAST+-2.14.1-gompi-2023a.eb)
```

### CODE REVIEW

**Comment by TopRichard** [comment]
bot: build repo:eessi.io-2023.06-software arch:x86_64/generic
bot: build repo:eessi.io-2023.06-software arch:x86_64/intel/haswell
bot: build repo:eessi.io-2023.06-software arch:x86_64/intel/skylake_avx512
bot: build repo:eessi.io-2023.06-software arch:x86_64/amd/zen2
bot: build repo:eessi.io-2023.06-software arch:x86_64/amd/zen3
bot: build repo:eessi.io-2023.06-software arch:aarch64/generic
bot: build repo:eessi.io-2023.06-software arch:aarch64/neoverse_n1
bot: build repo:eessi.io-2023.06-software arch:aarch64/neoverse_v1

**Comment by bedroge** [comment]
```
configure: creating cache config.cache
checking TeamCity build number... none
checking VCS revision... unknown
checking NCBI stable components' version... unknown
checking build system type... aarch64-unknown-linux-gnu
checking host system type... aarch64-unknown-linux-gnu
checking for a BSD-compatible install... /cvmfs/software.eessi.io/versions/2023.06/compat/linux/aarch64/usr/bin/install -c
checking for gcc... mpicc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether mpicc accepts -g... yes
checking for mpicc option to accept ISO C89... none needed
checking how to run the C preprocessor... mpicc -E
checking for aarch64-unknown-linux-gnu-gcc... aarch64-unknown-linux-gnu-gcc
checking whether we are using the GNU C compiler... yes
checking whether aarch64-unknown-linux-gnu-gcc accepts -g... yes
checking for aarch64-unknown-linux-gnu-gcc option to accept ISO C89... none needed
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... no
checking for suffix of object files... o
checking how to run the C preprocessor... aarch64-unknown-linux-gnu-gcc -E
checking whether we are using the GNU C++ compiler... yes
checking whether mpicxx accepts -g... yes
adjusted C   compiler: /cvmfs/software.eessi.io/versions/2023.06/software/linux/aarch64/generic/software/OpenMPI/4.1.5-GCC-12.3.0/bin/mpicc 
adjusted C++ compiler: /cvmfs/software.eessi.io/versions/2023.06/software/linux/aarch64/generic/software/OpenMPI/4.1.5-GCC-12.3.0/bin/mpicxx 
checking for grep that handles long lines and -e... /cvmfs/software.eessi.io/versions/2023.06/compat/linux/aarch64/bin/grep
checking for egrep... /cvmfs/software.eessi.io/versions/2023.06/compat/linux/aarch64/bin/grep -E
checking for ANSI C header files... no
checking for sys/types.h... no
checking for sys/stat.h... no
checking for stdlib.h... no
checking for string.h... no
checking for memory.h... no
checking for strings.h... no
checking for inttypes.h... no
checking for stdint.h... no
checking for unistd.h... no
checking minix/config.h usability... no
checking minix/config.h presence... no
checking for minix/config.h... no
checking whether it is safe to define __EXTENSIONS__... no
checking whether this system supports --with(out)-64... no
configure: error: cannot continue; please try different options
```
Okay, so we should just try..... different options :thinking:

**Comment by bedroge** [comment]
The easyconfig is setting `--with-64`, which seems to set `-m64`, and that's not supported on Arm. So we should fix this in the easyconfig.

**Comment by bedroge** [comment]
Fixed in https://github.com/easybuilders/easybuild-easyconfigs/pull/20784.

File: easystacks/software.eessi.io/2023.06/eessi-2023.06-eb-4.9.1-2023a.yml [hunk 1]
```diff
@@ -17,3 +17,6 @@ easyconfigs:
         from-pr: 20595
   - GATK-4.5.0.0-GCCcore-12.3.0-Java-17.eb
   - WhatsHap-2.2-foss-2023a.eb
+  - BLAST+-2.14.1-gompi-2023a.eb:
+      options:     
+        from-pr: 20751
```
Comment (bedroge) [inline easystacks/software.eessi.io/2023.06/eessi-2023.06-eb-4.9.1-2023a.yml:22]: Could you try to use PR 20784 and build this for one Arm target, just to verify that this solves the issue?
```suggestion
        from-pr: 20784
```

**Comment by TopRichard** [comment]
bot: build repo:eessi.io-2023.06-software arch:x86_64/generic
bot: build repo:eessi.io-2023.06-software arch:x86_64/intel/haswell
bot: build repo:eessi.io-2023.06-software arch:x86_64/intel/skylake_avx512
bot: build repo:eessi.io-2023.06-software arch:x86_64/amd/zen2
bot: build repo:eessi.io-2023.06-software arch:x86_64/amd/zen3

**Review by bedroge — APPROVED** [review]

**Pull request merged**